tools/aarnetfilesender/filesender.xml

```diff
@@ -34,10 +34,10 @@ mkdir filestosend &&
     ln -s "$file" "filestosend/${file.element_identifier}" &&
 #end for
 
-filesender -p -u '$apiuser' -a '$apikey' -b https://filesender.aarnet.edu.au/rest.php -f "galaxy-no-reply@usegalaxy.org.au" -r '$to' filestosend >$output 2>&1
+filesender -p -u '$apiuser' -a '$apikey' -b https://filesender.aarnet.edu.au/rest.php -f '$__user_email__' -r '$to' -s 'Data sent from Galaxy' filestosend >$output 2>&1
     ]]></command>
     <inputs>
-        <param name="data" type="data" format="txt,data" multiple="true" optional="false" label="Datasets to be sent:"/>
```

Comment:
@mthang do you know if `txt` is necessary here? I removed it and it still let me send a text file. Just because "Select a dataset of type data,txt" looks weird to my eye.

Reply:
@TomHarrop I would remove "txt" from the 'format. My understanding is the format data should already have the txt covered.